ddtrace/tracer.py

```diff
@@ -531,6 +539,23 @@ def start_span(
         self._hooks.emit(self.__class__.start_span, span)
         return span
 
+    def activate_distributed_header(self, headers, activate=True, propagator=None):
```

Comment:
```suggestion
    def extract(self, headers, activate=True, propagator=None):
```

Personally I think `extract` captures what we're doing here and then having `activate` as an argument makes it clear that you can extract and activate.